.circleci/config.yml

```diff
@@ -64,49 +63,46 @@ commands:
           key: _site-v1-{{ .Revision }}
           paths:
             - _site
-  compress:
-    description: "Compress"
+  deploy_gh_pages:
+    description: "Deploy to static branches"
+    parameters:
+      target_branch:
+        type: string
     steps:
+      - checkout
+      - attach_workspace:
+          at: _site
+      - run:
+          name: Tag build
+          command: echo "<< parameters.target_branch >> $(date)" > _site/version
       - run:
-          name: "Compress"
+          name: Install and configure dependencies
           command: |
-            pushd _site/
-            tar -cvf artifact.tar *
-            mv artifact.tar ${OLDPWD}/
+            curl -sL https://deb.nodesource.com/setup_12.x | sudo -E bash -
+            sudo apt install nodejs
+            mkdir "/home/circleci/.npm-packages"
+            npm config set prefix "/home/circleci/.npm-packages"
+            npm install -g gh-pages@2.0.1
```

Comment:
@gokula-krishna-dev you can install gh-pages locally

Reply:
That would add dependency to the project which we should avoid